content/applications/sales/crm/optimize/lead_enrichment.rst

```diff
@@ -0,0 +1,113 @@
+--------------------------
+
+During set up, if :guilabel:`Enrich all leads automatically` was selected, then no action needs to
+be taken by the user to enrich the lead. A scheduled action runs automatically, every sixty minutes,
+and enrichment occurs on leads, after a remote database is contacted.
+
+.. tip::
+   To access the cron that runs for the automatic lead enrichment, activate :ref:`developer mode
+   <developer-mode>`, and navigate to :menuselection:`Settings app --> Technical menu --> Automation
+   --> Scheduled Actions`. In the :guilabel:`Search...` type in `partner autocomplete`. Click into
+   the result labeled :guilabel:`Partner Autocomplete Sync with remote DB`, and make any necessary
+   adjustments. In the :guilabel:`Execute Every` field, enter a value greater than or equal to five
+   minutes.
+
+.. example::
+   The following is an example of a successful autocomplete of lead enrichment data:
+
+   .. image:: lead_enrichment/lead-enrichment-data.png
+      :align: center
+      :alt: Chatter showing lead enrichment data.
+
+Manually enrich leads
+---------------------
+
+If the :guilabel:`Enrich leads on demand only` option was selected when activating :guilabel:`Lead
+Enrichment`, each lead that a user wishes to enrich will need to be manually enriched. This is
+achieved by clicking the :guilabel:`Enrich` button in the top menu of the lead.
+
+The same information will be retrieved at the same :abbr:`IAP (In-App Puchase)` credit cost (one per
+enrichment). This method of enrichment is useful when every lead does not need to be enriched or
+cost is an issue.
+
+.. image:: lead_enrichment/manual-enrichment.png
+   :align: center
+   :alt: Manual enrich button feature highlighted on the CRM lead.
+
+.. tip::
+   Manually enrich leads in bulk using the *list* view. First navigate to the user's *CRM* pipeline
+   (:menuselection: CRM app --> Sales --> My Pipeline`) and click the list view (:guilabel:`☰ (three
+   horizontal lines)` icon). Next, tick the checkboxes for the leads that should be manually
+   enriched. Finally, click the :guilabel:`⚙️ Actions` icon, and select :guilabel:`Enrich`.
+
+Pricing
+=======
+
+Lead enrichment is an *In-App Purchase* feature and each enriched lead costs one credit.
+
+.. note::
+   See here for full pricing information: `Lead Generation by Odoo IAP
+   <https://iap.odoo.com/iap/in-app-services/273>`_.
+
+To buy credits, navigate to :menuselection:`CRM app --> Configuration --> Settings`. In the
+:guilabel:`Lead Generation` section, under the :guilabel:`Lead Enrichment` feature, click on
+:guilabel:`Buy Credits`.
+
+.. image:: lead_enrichment/buy-lead-enrichment-credits-setting.png
+   :align: center
+   :alt: Buy credits from the lead enrichment settings.
+
+Credits and balances may also be purchased by navigating to :menuselection:`Settings --> General
+Settings`. In the :guilabel:`In-App Purchases` section, under the :guilabel:`Odoo IAP` feature,
+click on :guilabel:`View My Services`.
```

Comment:
```suggestion
Credits and balances may also be purchased by navigating to the :menuselection:`Settings app`. In the :guilabel:`Contacts` section, under the :guilabel:`Odoo IAP` feature,
click on :guilabel:`View My Services`.
```
This section appears as 'Contacts' in the 15.0 runbot I'm using for the review. Also, it is my understanding that, when the 'Settings' app is opened, it already reveals the 'General Settings' -- so I dont' believe it's necessary for readers to navigate any further once they open the application